src/Quartz/Impl/Matchers/StringOperator.cs

```diff
@@ -20,13 +20,17 @@
 #endregion
 
 using System;
+using System.Runtime.Serialization;
 
 namespace Quartz.Impl.Matchers
 {
     /// <summary>
     /// Operators available for comparing string values.
     /// </summary>
+#if BINARY_SERIALIZATION
     [Serializable]
+#endif // BINARY_SERIALIZATION
+    [DataContract]
     public abstract class StringOperator : IEquatable<StringOperator>
```

Comment:
as operators are really not persisted, we could probably drop serialization support altogether now